cvat-ui/src/reducers/server-api-reducer.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2024 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { BoundariesActions, BoundariesActionTypes } from 'actions/boundaries-actions';
+import { ServerAPIActions, ServerAPIActionTypes } from 'actions/server-actions';
+import { ServerAPIState } from '.';
+
+const defaultState: ServerAPIState = {
+    schema: null,
+    fetching: false,
+    initialized: false,
+    isRegistrationEnabled: true,
+    isBasicLoginEnabled: true,
+    isPasswordResetEnabled: true,
+    isPasswordChangeEnabled: true,
```

Comment:
Do we really want the configuration to be true for everything by default? As far as I remember, before this PR reset/change passwords were disabled by default.